OpenRA.Utility/Program.cs

```diff
@@ -62,7 +62,8 @@ static void Main(string[] args)
 			{
 				var command = (IUtilityCommand)Activator.CreateInstance(commandType);
 				var kvp = new KeyValuePair<Action<ModData, string[]>, Func<string[], bool>>(command.Run, command.ValidateArguments);
-				actions.Add(command.Name, kvp);
+				if (!actions.ContainsKey(command.Name))
+					actions.Add(command.Name, kvp);
```

Comment:
The trick is doing that in a way that doesn't require the mod choices to be hardcoded (because that then defeats the purpose).